src/List.js

```diff
@@ -3,14 +3,23 @@ import { Link } from 'react-router-dom';
 
 export default class List extends Component {
 
-    
     render() {
-       
-        return (
-            <div>
-                {console.log(this.props.pokemon.results)}
-                <p>this is a LIST of all the pokemonz!!!</p> 
+        const pokemonArray = this.props.pokemon
```

Comment:
Looks like this is undefined. I would console.log this.props and see what's coming through and maybe readjust your strategy based on that!